rhoast_functions/src/rholang/mod.rs

```
@@ -9,6 +9,24 @@ pub fn parse_simple_deploy(text: &String) -> Result<String, Box<dyn Error>> {
     Ok(compiled)
 }
 
+fn parse_simple(rholang: &String, arg: &String, text: &String) -> Result<String, Box<dyn Error>> {
+    let reg = Handlebars::new();
+    let compiled = reg.render_template(&rholang, &json!({ arg: &text }))?;
+    Ok(compiled)
+}
+
+pub fn parse_multiple_args(
+    rholang: &String,
+    args: &BTreeMap<String, String>,
+) -> Result<String, Box<dyn Error>> {
+    let mut rtc: String = rholang.to_string();
+    for a in args.keys() {
+        let val = args.get(a).unwrap();
```

Can we return the error here instead?